src/app/app.component.ts

```diff
@@ -237,7 +200,9 @@ export class AppComponent implements OnInit, OnDestroy {
 	}
 
 	removePackageName(packageName: string): void {
-		this.packageNames.removeValue(packageName);
+		const loadedPackageNames = this.packageNames.getValue();
+		const newValue = loadedPackageNames.filter((value) => value !== packageName);
+		this.packageNames.set(newValue);
```

Comment:
was there something wrong with `this.packageNames.removeValue(packageName);`?

Reply:
reverted in last commit